### PR TITLE
travis fixes and test

### PR DESCRIPTION
**Pull request by sebix**
* `Event()` constructor takes optional argument: instance of report, fields `raw`, `feed.(name|url)`, `time.observation` are copied if existing. reduces duplicate code.
* Changes for travis and Python 3 requirements
* Tests requiring local dbs are skipped unless these files exits
* using `setup_travis.py` sudo and `/opt/intelmq` are no longer needed.
* documentation on bots started (with experts)
* removed EBNF crap from Data-Harmonization docs.
* New field `classification.identifier` plus docs
* BOTS: switched most sources to HTTPS, updated description for some
* asn-lookup ignores IPv6 until fixed upstream -> no errors and stopping bots.
* ripencc bot has two more sources to get abuse contacts for AS and IP
* Fix in cache for non-string values
* fixed timezone handling for `harmonization.DateTime.from_timestamp`

### CODE REVIEW

**Comment by sebix** [comment]
Finally, Tests for 2.7 are passing! Damn timezones!!!!

**Comment by SYNchroACK** [comment]
great!

**Comment by aaronkaplan** [comment]
Congrats. 

---
Mobile

File: intelmq/bots/BOTS [hunk 1]
```diff
@@ -1,56 +1,56 @@
 {
     "Collector": {
         "Abuse.ch Feodo Tracker Domains": {
-            "description": "Abuse.ch Feodo Tracker Domains is the bot responsible to get the report from source of information.",
```
Comment (SYNchroACK) [inline intelmq/bots/BOTS]: The description has a pattern: starts always with the name of the bot, e.g. 'Abuse.ch Feodo Tracker Domains'

**Comment by SYNchroACK** [comment]
> * `Event()` constructor takes optional argument: instance of report, fields `raw`, `feed.(name|url)`, `time.observation` are copied if existing. reduces duplicate code.

I think we should keep with other approach. Its clear and don't have hidden functionality. Whats your feedback?

> * Changes for travis and Python 3 requirements
> * Tests requiring local dbs are skipped unless these files exits
> * using `setup_travis.py` sudo and `/opt/intelmq` are no longer needed.
> * documentation on bots started (with experts)
> * removed EBNF crap from Data-Harmonization docs.
> * New field `classification.identifier` plus docs
> * BOTS: switched most sources to HTTPS, updated description for some
> * asn-lookup ignores IPv6 until fixed upstream -> no errors and stopping bots.
> * ripencc bot has two more sources to get abuse contacts for AS and IP
> * Fix in cache for non-string values
> * fixed timezone handling for `harmonization.DateTime.from_timestamp`

File: intelmq/bots/BOTS (hunk 1 above)
Comment (sebix) [inline intelmq/bots/BOTS]: -----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

On 09/08/2015 06:47 PM, Tomás Lima wrote:
>
> The description has a pattern: starts always with the name of the bot,
e.g. 'Abuse.ch Feodo Tracker Domains'
>
And is absolutely useless. Sorry, but that's my opinion on this topic.
The name is known from the key, just one line above. I assume everyone
reading this file can read both lines. Also, the file repeats the
definition of the bots types is repeated 116 times. And in the docs,
where it belongs to.

>|> Event()| constructor takes optional argument: instance of report, fields |raw|, |feed.(name|url)|,
|time.observation| are copied if existing. reduces duplicate code.

> I think we should keep with other approach. Its clear and don't have
hidden functionality. Whats your feedback?

Well, I'm for the change :) It's *exactly* the same code for *every*
parser. It reduces code in the bots, the the attention of the reader
(dev) is focused on the relevant parts, i.e. all the data handling
except those just copied - they are not relevant for the parser code.
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2.0.22 (GNU/Linux)

iQIcBAEBCAAGBQJV7yPKAAoJEBn0X+vcm0Y7lhAQALZ3uggU8XwnD7ja3f1u5Ccc
E23BgUiYqv2nmV7d/AIYpLM7giqWo0MZ8F8aoZU0J+Z18bh9Pg/Noe68AZ+zs/Ld
5qtO1gqinCNcTQFSu0C7qFpfi8Vmpyh/8Xz7i1wDwISH4LTNq0F6ZulW4ISTL80g
W9VDXf8DCl0X1uRyvdkqw0D3WBKPjXKHaHu5QM9SgWFLmseNu77CYPCUYHa2WHG0
5nIbuIda5bhPR6msVutW/t5QWd652EanKGDYoxygdHmOK4epTiOHmM3b8+vcvIQs
7oQFv/noACf0Dh2/UjXGNsJWPq8gi2xYInPYC9SbIU/dgoh+B9GvY+ckMIyenRDk
jPlntREnrXGYQS8Kj1If5oWTC9bQJasZlXPvP/M/sX7V/xdocHxHlUGuce1+fk6V
c4oS8/FN6pRpjEning18E9hTUFF+moshLiEBf9QkDEGYewlU8RpW8oJYEGsvPtpd
PGyKV8zjPkw//VolGjGUmwVC1ML/v7pkoU3kH5IWpKMRp2eRdGz3rGVn5OBNDZbr
ABA6Qy2DmLytWc7ZyBTQeeCYB+ytmRJq5TFvHxDs4ICZWmCdg3PJkDhrp7EK5O5o
YCrvrHx4WpRuOVynLtiUrCnD3laXklfjA/vhiBRBKlqjeoySN5xwy4KPozkopLrE
AJNzQhNVCYBiB/Xcllox
=WCGC
-----END PGP SIGNATURE-----

Comment (aaronkaplan) [inline intelmq/bots/BOTS]: +1

---
Mobile

**Pull request merged**